README.md

```diff
@@ -20,6 +20,7 @@ to something other than a raw file it should be marked with an asterisk.*
 | [`centerdoc`](plugins/centerdoc.lua?raw=1)                                                                     | Centers document's content on the screen and adds zen mode support *([screenshot](https://user-images.githubusercontent.com/3920290/82127896-bf6e4500-97ae-11ea-97fc-ba9a552bc9a4.png))*                                                                                                        |
 | [`colorpreview`](plugins/colorpreview.lua?raw=1)                                                               | Underlays color values (eg. `#ff00ff` or `rgb(255, 0, 255)`) with their resultant color. *([screenshot](https://user-images.githubusercontent.com/3920290/80743752-731bd780-8b15-11ea-97d3-847db927c5dc.png))*                                                                                  |
 | [`console`](https://github.com/franko/console)                                                                 | A console for running external commands and capturing their output *([gif](https://user-images.githubusercontent.com/3920290/81343656-49325a00-90ad-11ea-8647-ff39d8f1d730.gif))*                                                                                                               |
+| [`exterm`](https://github.com/ShadiestGoat/lite-xl-exterm)                                                                 | Allows to open an external console in current project directory |
```

Comment:
Would you mind adding this asterisk?
```suggestion
| [`exterm`](https://github.com/ShadiestGoat/lite-xl-exterm)\*                                                                 | Allows to open an external console in current project directory |
```

Reply:
No problem, sec. I wantned to ask, what do the asterisks signify? I couldn't find the key anywhere?

Reply:
It's written in the README: `If a plugin's link resolves to something other than a raw file it should be marked with an asterisk.`